settings/DevHome.Settings/ViewModels/AboutViewModel.cs

```diff
@@ -1,21 +1,32 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Collections.ObjectModel;
 using CommunityToolkit.Mvvm.ComponentModel;
 using DevHome.Common.Extensions;
 using DevHome.Common.Services;
+using DevHome.Settings.Models;
 using Microsoft.UI.Xaml;
 
 namespace DevHome.Settings.ViewModels;
 
-public partial class AboutViewModel : ObservableObject
+public partial class AboutViewModel : BreadcrumbViewModel

```

Comment:
This inheritance is not very accurate since the AboutViewModel `IS` not a BreadcrumbViewModel